dali/operators/input/CMakeLists.txt

```diff
@@ -14,5 +14,13 @@
 
 # Get all the source files and dump test files
 collect_headers(DALI_INST_HDRS PARENT_SCOPE)
-collect_sources(DALI_SRCS PARENT_SCOPE)
-collect_test_sources(DALI_TEST_SRCS PARENT_SCOPE)
+
+if (BUILD_FFMPEG)
+    list(APPEND DALI_INST_HDRS "${CMAKE_CURRENT_SOURCE_DIR}/video_input.h")
+    list(APPEND DALI_OPERATOR_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/video_input.cc")
+endif (BUILD_FFMPEG)
+
+set(DALI_OPERATOR_SRCS ${DALI_OPERATOR_SRCS} PARENT_SCOPE)
+set(DALI_INST_HDRS ${DALI_INST_HDRS} PARENT_SCOPE)
+
+collect_test_sources(DALI_OPERATOR_TEST_SRCS PARENT_SCOPE)
```

Comment:
Will that work if `BUILD_FFMPEG` is OFF?

Reply:
Wouldn't `input/video` and conditional adding of a whole video dir be cleaner?

Reply:
This directory will also have a GPU variant with `BUILD_NVDEC` option, so I believe it's better with particular `if`.

Fixed the `collect_test_sources` part.